src/services/Review/updateReview.services.ts

```diff
@@ -0,0 +1,20 @@
+import { Request } from "express";
+import { ErrorHandler } from "../../errors";
+import { CourseReviewRepository } from "../../repositories";
+
+const updateReviewService = async (req: Request) => {
+  try {
+
+    const { id } = req.params;
+    const objectCourseReview = new CourseReviewRepository();
+    await objectCourseReview.updateReview(id,req.validated);
+    return await objectCourseReview.findOneOrFail(id);
```

Comment:
Não é um problema, mas ao invés do `Repo.update()` no método `updateReview()`, pode ser usado o `Repo.save()`, que já retorna a entidade atualizada. Dessa forma não seria necessário rodar um outro método de busca.
https://stackoverflow.com/questions/47792808/typeorm-update-item-and-return-it